dracut/nfsrepo-cleanup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
Could you please add explanation why is this needed here as a comment?

You can just copy the description of this PR here. It will be really valuable in the future.

Reply:
Thanks Jiri, I will update it.

Reply:
Thanks a lot!